Modix.Services/Core/AuthorizationService.cs

```diff
@@ -405,6 +427,20 @@ public Task<IReadOnlyCollection<AuthorizationClaim>> GetGuildUserClaimsAsync(IGu
             return LookupPosessedClaimsAsync(guildUser.GuildId, guildUser.RoleIds, guildUser.Id, filterClaims);
         }
 
+        /// <inheritdoc />
+        public Task<IReadOnlyCollection<AuthorizationClaim>> GetGuildUserClaimsAsync(ulong user, ulong guild, IReadOnlyList<ulong> roles, params AuthorizationClaim[] filterClaims)
+        {
+            if (user == default)
+                return Task.FromException<IReadOnlyCollection<AuthorizationClaim>>(new ArgumentNullException(nameof(user)));
+
+            if (user == CurrentUserId)
+                return Task.FromResult(CurrentClaims);
+
+            if ((user == _discordSocketClient.CurrentUser.Id))
```

Comment:
Small regression here, the version of `GetGuildUserClaimsAsync` that takes an `IGuildUser` grants every claim to anyone with the Administrator permission - this method should as well